### PR TITLE
[vm-repair] Bug fix for disk copy V2 to V1 and Adding preview parameter

### PR DESCRIPTION
**Pull request by haagha**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vm-repair

**Comment by zhoxing-ms** [comment]
@haagha Could you add some tests for this case?

**Comment by zhoxing-ms** [comment]
Please write those changes into the `HISTORY.rst`

By the way, do you need to release a new version for these changes? If so, please also update the `setup.py`

**Comment by zhoxing-ms** [comment]
@haagha Any update?

**Comment by haagha** [comment]
@zhoxing-ms  Yes i updated the history and build version. Will try to fix the CI failures.

**Comment by haagha** [comment]
@zhoxing-ms Can you review this, thanks!

File: src/vm-repair/azext_vm_repair/custom.py [hunk 1]
```diff
@@ -59,9 +67,10 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
         else:
             os_image_urn = _fetch_compatible_windows_os_urn(source_vm)
             os_type = 'Windows'
+
         # check hyperv Generation
-        if enable_nested:
-            _check_hyperV_gen(source_vm)
+        # if enable_nested and (is_gen2 == 2):
+        #    raise SkuDoesNotSupportHyperV('Cannot support V2 HyperV generation. Please run command without --enabled-nested')
```
[the remark this answers is not in the document]
Comment (haagha) [reply src/vm-repair/azext_vm_repair/custom.py]: removed.

**Comment by haagha** [comment]
@zhoxing-ms  Removed comment and added test.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**